backend/connection_manager.py

```
@@ -0,0 +1,72 @@
+    async def send_to_drawing(self, message: dict, drawing_id: str):
+        """Отправляет JSON-данные всем, кто открыл конкретный чертеж."""
+        if drawing_id in self.active_connections:
+            connections = self.active_connections[drawing_id]
+
+            # ЛОГ: Самый важный этап — подтверждение, что бэкенд видит получателей
+            logger.info(f"Рассылка данных в WS для {drawing_id}. Получателей: {len(connections)}")
+
+            for connection in connections:
+                try:
+                    await connection.send_json(message)
+                except Exception as e:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)